packages/j/jose.json

```diff
@@ -57,6 +71,12 @@
         "files": [
           "**/*.js"
         ]
+      },
+      {
+        "basePath": "dist/webapi",
```

Comment:
Is this directory going to have files in it that have names that collide with files in dist/browser? As you're using `basePath` here, we're going to pull up all files from this subdirectory into the root

Reply:
the two basepaths are exclusive to their respective release lines

`<  v6` uses `dist/browser`
`>= v6` uses `dist/webapi`

Reply:
Ah, wonderful!